re.sonny.Workbench.Devel.json

```diff
@@ -71,6 +71,18 @@
         }
       ]
     },
+    {
+      "name": "libmanette",
+      "buildsystem": "meson",
+      "config-opts": ["-Dgudev=disabled"],
```

Comment:
is the `config-opts` what fixed it in flatpak?

Reply:
Yes!

Reply:
https://gitlab.gnome.org/GNOME/gnome-build-meta/-/merge_requests/2217